Jellyfin.Plugin.Tvdb/Providers/TvdbMissingEpisodeProvider.cs

```diff
@@ -94,14 +97,14 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
-        private static bool IsValidEpisode(EpisodeRecord? episodeRecord)
+        private static bool IsValidEpisode(EpisodeBaseRecord? episodeRecord)
         {
-            return episodeRecord?.AiredSeason != null && episodeRecord.AiredEpisodeNumber != null;
+            return episodeRecord?.SeasonNumber != null && episodeRecord?.Number != null;
```

Comment:
Hey, this was part of the old code when i was migrating from v3 to v4. I didn't remove it just in case if it broke anything. Think this was used when TVDb data was dirtier then current. As far as I can tell, it is no longer needed since according to the schema from TVDb, it will not be null.